editor/plugins/animation_blend_space_1d_editor.cpp

```diff
@@ -596,7 +596,9 @@ void AnimationNodeBlendSpace1DEditor::_notification(int p_what) {
 			String error;
 
 			if (!tree->is_active()) {
-				error = TTR("AnimationTree is inactive.\nActivate to enable playback, check node warnings if activation fails.");
+				error = AnimationTree::get_editor_inactive_error_message();
+			} else if (!tree->is_enabled()) {
+				error = AnimationTree::get_editor_disabled_error_message();
 			} else if (tree->is_state_invalid()) {
 				error = tree->get_invalid_state_reason();
 			}
```

Comment:
What I meant is that you can put this whole block in a method. This part is literally the same every time.